cmd/init.go

```diff
@@ -19,304 +19,243 @@ import (
 				return err
 			}
-
-			if err := p.Run(); err != nil {
-				fmt.Printf("%v\n", err)
+			// possibly no stores exist yet - create one
+			if err := params.createStore(); err != nil {
 				return err
 			}
-
-			cmd.Println()
-
-			table := tablewriter.CreateTable()
-			table.UTF8Box()
-			table.AddTitle("Generated NKeys Location")
-			table.AddHeaders("Name", "Kind", "Location")
-			printed := false
-			for _, c := range p.Containers() {
-				if c.create && c.generated {
-					printed = true
-					table.AddRow(c.name, c.kind.String(), AbbrevHomePaths(c.keyPath))
-					if c.kind == nkeys.PrefixByteUser {
-						s, err := GetStore()
-						if err == nil {
-							ctx, err := s.GetContext()
-							if err == nil {
-								creds := ctx.KeyStore.GetUserCredsPath(p.account.name, p.user.name)
-								if creds != "" {
-									table.AddRow("", "creds file", AbbrevHomePaths(creds))
-								}
-							}
-						}
+			if err := RunAction(cmd, args, &params); err != nil {
+				return fmt.Errorf("init failed: %v", err)
+			}
+			cmd.Printf("Success!! created a new operator, account and user named %q.\n", params.Name)
+			cmd.Printf("User creds file stored in %q\n\n", params.User.CredsPath)
+
+			if prompted {
+				ok, err := cli.PromptBoolean("deploy account to a managed operator", false)
+				if ok {
+					params.Deploy, err = cli.Prompt("enter operator url", "", true, cli.URLValidator("http", "https"))
+					if err != nil {
+						return err
 					}
 				}
 			}
 
-			if printed && !QuietMode() {
-				cmd.Println(cli.Wrap(70, "Project initialization generated NKeys.",
-					"These keys should be treated as secrets.", "You can move the directory,",
-					"and reference them from the", fmt.Sprintf("`$%s`", store.NKeysPathEnv),
-					"environment variable. To remind yourself of current environment",
-					"configuration type `nsc env` while in a project directory.\n"))
-				cmd.Println(table.Render())
+			if params.Deploy != "" {
+				var deploy DeployCmdParams
+				deploy.AccountContextParams.Name = params.Name
+				deploy.url = params.Deploy
+
+				if err := RunAction(cmd, args, &deploy); err != nil {
+					return fmt.Errorf("deploy %q failed - %v", params.Name, err)
+				}
+				cmd.Printf("Success!! deployed %q to operator %q\n", params.Name, deploy.claim.Name)
 			}
 
-			if p.operator.create {
-				cmd.Println("Success! - initialized environment directory")
+			if params.Deploy == "" {
+				cmd.Printf("\n\nTo run a local server using this configuration, enter:\n")
+				cmd.Printf("> nsc generate config --mem-resolver --config-file <path/server.conf>\n")
+				cmd.Printf("start a nats-server using the generated config:\n")
+				cmd.Printf("> nats-server -c <path/server.conf>\n\n")
+
+				cmd.Printf("Or deploy your account to a managed service enter:\n")
+				cmd.Printf("> nsc deploy --url https://jwt.ngs.local:6060/jwt/v1/operator\n\n")
 			}
 
+			cmd.Printf("To listen for messages enter:\n")
+			cmd.Printf("> nsc tools sub \">\"\n")
```

Comment:
maybe this is where we can say the operator JWT defaults to pointing at a local server